timeseries/tests/smoketests/test_all_models.py

```diff
@@ -138,6 +138,7 @@ def test_all_models_can_handle_all_covariates(
 
 
 @pytest.mark.parametrize("freq", DEFAULT_SEASONALITIES.keys())
+@pytest.mark.skipif(parse_version(pd.__version__) < parse_version("2.1"), reason="Skipping test for pandas < 2.1")
```

Comment:
Would need @shchur to advise on this specific change

Reply:
@prateekdesai04 thank you for identifying the root cause of the problems! I narrowed down the conditions under which the tests are skipped in #4051 (needed to create a new PR because I'm still not sure how to run platform tests from forks 😅).

Reply:
Should I close this PR ?